metadata_store_scripts/bootstrap.sql

```diff
@@ -546,3 +545,38 @@ BEGIN
         END
 -- End stored procedure definition
 END;
+-- source: V2024.10.16.0__add_azuresql_to_azuresql_support
+DELETE FROM adf_type_mapping WHERE dataset = 'AZURESQL';
```

Comment:
After performing the rename of the new migration file, please re-run the script that updates this file and include the updates in the PR.